python/lsst/ts/standardscripts/maintel/setup_mtcs.py

```diff
@@ -0,0 +1,380 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["SetupMTCS"]
+
+import asyncio
+
+# import logging
+# import os
+# import sys
+import yaml
+
+import pandas as pd
+
+from lsst.ts import salobj, utils
+from lsst.ts.observatory.control.maintel.mtcs import MTCS
+
+
+class SetupMTCS(salobj.BaseScript):
+    """
+    Setup MTCS components so they are ready for operation.
+
+    The script starts with MTPtg, MTMount, and MTRotator.
+    Then, checks if the camera-cable-wrap (CCW) is following the Rotator.
+    It also checks the telemetry from these components.
+
+    After that, it points the mount to the Zenith so we can raise M1M3 safely.
+    M1M3 is enabled, raised and the corrections are enabled and reset.
+    We do the same for M2, Camera Hexapod and Rotator Hexapod.
+
+    Finally, it enables MTAOS, MTDomeTrajectory and MTDome.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    (none)
+
+    **Details**
+
+    This script starts them in the following order:
+      - mtptg
+      - mtmount
+      - mtrotator
+      - mtm1m3
+      - mtm2
+      - mthexapod_1 (camera hexpapod)
+      - mthexapod_2 (m2 hexapod)
+      - all the other components (mtaos / mtdome / mtdometrajectory)
+
+    """
+
+    def __init__(self, index=1):
+        super().__init__(
+            index=index,
+            descr="Setup MTCS components for operations.",
+        )
+        self.config = None
+        self.mtcs = MTCS(domain=self.domain, log=self.log)
+
+    @classmethod
+    def get_schema(cls):
+        schema = """
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: https://github.com/lsst-ts/ts_externalscripts/blob/master/python/lsst/ts/externalscripts/maintel/setup_mtcs.py
```

Comment:
This line is too long.. Also, not is it referring to externalscripts, this is standardscripts